scripts/analysis.py

```diff
@@ -56,188 +25,85 @@ def get_agent_ids(cursor, facility):
     cursor: cursor
         cursor for sqlite3
     facility: str
-        name of facility type
+        name of facility type as described in spec
 
     Returns
     -------
-
     sink_id: list
-        list of all the sink agentId values.
+        list of all the sink agentId values as string.
     """
+    agents = cursor.execute("SELECT * FROM agententry WHERE spec LIKE '%" +
+                            facility + "%' COLLATE NOCASE").fetchall()
 
-    cur = cursor
-    agent_id = []
-    agent = cur.execute("select * from agententry where spec like '%"
-                        + facility + "%'").fetchall()
+    return list(str(agent[1]) for agent in agents)
 
-    for ag in agent:
-        agent_id.append(ag[1])
-    return agent_id
 
-
-def get_waste_id(resource_list):
-
-    """ Gets waste id from a resource list
+def get_prototype_id(cursor, prototype):
+    """ Returns agentid of a prototype
 
     Parameters
-    ---------
-    resource_list: list
-        list fetched from the resource table.
+    ----------
+    cursor: sqlite cursor
+        sqlite cursor
+    prototype: str
+        name of prototype
 
     Returns
     -------
-    waste_id: list
-        list of qualId for waste
+    agent_id: list
```

Comment:
If you are referring to the get_waste_id function, that has been removed.
If not, can you point to the function that you're referring to please?

Thank you.